pkg/operator/ceph/cr_manager.go

```diff
@@ -156,11 +157,19 @@ func (o *Operator) startCRDManager(context context.Context, mgrErrorCh chan erro
 
 	// Set up a manager
 	mgrOpts := manager.Options{
-		LeaderElection:     false,
-		Namespace:          o.config.NamespaceToWatch,
```

Comment:
As discussed, we need to confirm that there is the ability to restrict the operator to watch for CRs only in the current namespace. This setting was controlled by `ROOK_CURRENT_NAMESPACE_ONLY`. By default all namespaces were watched, but some clusters still need the ability to restrict the watch to a single namespace. I believe the RBAC doesn't actually restrict the watcher for Rook CRs.